va/va_vpp.h

```diff
@@ -871,11 +1011,19 @@ typedef struct _VAProcPipelineParameterBuffer {
 
     VAProcColorProperties  output_color_properties;
 
+    /**
+     * \brief Output High Dynamic Metadata.
+     *
+     * If output_metadata is NULL, then output default to SDR.
+     */
+    VAHdrMetaData          *output_hdr_metadata;
+    VAProcMode             processing_mode;
+
     /** \brief Reserved bytes for future use, must be zero */
     #if defined(__AMD64__) || defined(__x86_64__) || defined(__amd64__)|| defined(__LP64__)
-    uint32_t                va_reserved[VA_PADDING_LARGE - 13];
+    uint32_t                va_reserved[VA_PADDING_LARGE - 15];
```

Comment:
please double check the reserved byte size, make sure the struct size is not changed